core/modules/search/search.admin.inc

```diff
@@ -137,6 +137,15 @@ function search_admin_settings($form, &$form_state) {
     '#description' => t('Only one type of item will be searched by default. This selection will determine what appears at <a href="@search">@search</a>.', array('@search' => url('search', array('absolute' => TRUE)))),
   );
 
+  /* Add checkbox for partial word search */
+  $search_partial_words = $config->get('search_partial_words_enable');
+  $form['active']['search_partial_words_enable'] = array(
+    '#title' => t('Enable partial word search'),
+    '#type' => 'checkbox',
+    '#default_value' => $config->get('search_partial_words_enable'),
+    '#description' => t('Check setting to enable partial word search, uncheck for full word only search.')
```

Comment:
This description can probably be removed. It doesn't add additional value to the label alone.